docs/examples/popover/dontCaret.js

```diff
@@ -0,0 +1,161 @@
+    title: 'Fashion',
+    alt: 'Thumbnail image: a white dress with red flowers',
+  },
+  {
+    url: 'https://i.ibb.co/swC1qpp/IMG-0494.jpg',
+    title: 'Food',
+    alt: 'Thumbnail image: a paella with shrimp, green peas, red peppers and yellow rice',
+  },
+  {
+    url: 'https://i.ibb.co/PFVF3JH/photo-1583847268964-b28dc8f51f92.webp',
+    title: 'Home',
+    alt: 'Thumbnail image: a living room with a white couch, two paints in the wall and wooden furniture',
+  },
+];
+
+function List({
+  handleImageTap,
+  title,
+}: {|
+  handleImageTap: (imageTitle: string) => void,
+  title: string,
+|}) {
+  return (
+    <Flex direction="column" gap={{ column: 4, row: 0 }}>
+      <Text color="default" size="100">
+        {title}
+      </Text>
+      <Flex direction="column" gap={{ column: 4, row: 0 }}>
+        {images.map(({ alt, title: imageTitle, url }) => (
+          <TapArea
+            key={title}
+            onTap={() => {
+              handleImageTap(imageTitle);
+            }}
+          >
+            <Flex gap={{ row: 2, column: 0 }} alignItems="center">
+              <Box height={50} width={50} overflow="hidden" rounding={2}>
+                <Mask rounding={2}>
+                  <Image
+                    alt={alt}
+                    color="rgb(231, 186, 176)"
+                    naturalHeight={50}
+                    naturalWidth={50}
+                    src={url}
+                  />
+                </Mask>
+              </Box>
+              <Text align="center" color="default" weight="bold">
+                {imageTitle}
+              </Text>
+            </Flex>
+          </TapArea>
+        ))}
+      </Flex>
+    </Flex>
+  );
+}
+
+export default function Example(): Node {
+  const [open, setOpen] = useState(false);
+  const [selectedBoard, setSelectedBoard] = useState('Fashion');
+  const anchorRef = useRef();
+
+  const handleImageTap = (imageTitle: string) => {
+    setSelectedBoard(imageTitle);
+    setOpen(false);
+  };
+
+  return (
+    <Box padding={6}>
+      <Flex alignItems="center" gap={2} justifyContent="center" width="100%">
+        <Button
+          accessibilityHaspopup
+          accessibilityExpanded={open}
+          accessibilityControls="main-example"
+          color="white"
+          iconEnd="arrow-down"
+          onClick={() => setOpen(!open)}
+          ref={anchorRef}
+          size="lg"
+          selected={open}
+          text={selectedBoard}
+        />
+
+        <Button color="red" onClick={() => {}} size="lg" text="Save" />
+      </Flex>
+
+      {open && (
+        <Layer>
+          <Popover
+            accessibilityLabel="Save to board"
+            anchor={anchorRef.current}
+            id="main-example"
+            idealDirection="down"
+            onDismiss={() => setOpen(false)}
+            positionRelativeToAnchor={false}
+            size="xl"
+            showDismissButton
+            showCaret
+          >
+            <Box width={360}>
```

Comment:
I'd to 250 here so it displays nicely

Reply:
I split the difference and went with 300 (250 seemed a tad narrow)